eth/catalyst/api.go

```diff
@@ -58,9 +58,9 @@ const (
 	// to be reprocessed again.
 	invalidBlockHitEviction = 128
 
-	// invalidTipsetsCap is the max number of recent block hashes tracked that
+	// invalidTippetsCap is the max number of recent block hashes tracked that
 	// have lead to some bad ancestor block. It's just an OOM protection.
-	invalidTipsetsCap = 512
+	invalidTippetsCap = 512
```

Comment:
Any reason for the change? AFAIK the tipset here is correct referring the chain tip set

Reply:
Oh, this is a wrong hint from golangci-lint(catalyst).
Reverted.